en/ros/README.md

```diff
@@ -11,19 +11,20 @@ ROS is only officially supported on Linux platforms.
 
 # Robotics using ROS 2
 
-[ROS 2](https://index.ros.org/doc/ros2/) is the newest version of ROS 2, adapting most of the changes that occurred in ROS (1) over the last years and bringing the best of ROS (1),
-while at the same time improving some things that weren't so go good.
+[ROS 2](https://index.ros.org/doc/ros2/) is the newest version of ROS.
+It captures most of the learnings and recently added features of ROS (1), improving a number of flaws of theearlier version.
 
-The interface between ROS 2 and PX4 is possible through the [*microRTPS* bridge](../middleware/micrortps.md), a bridge established between the PX4 internals, using the uORB Pub/Sub
-architecture, and correspondent IDL types, that are used to generate, not only serialization and deserialization code, but also the ROS 2 messages and typesupport, required to use
-the equivalents of the uORB messages in the ROS 2 development workflows and nodes.
+The translation layer between ROS 2 and PX4 is software known as the [*microRTPS* bridge](../middleware/micrortps.md).
+This provides a bridge between PX4 UORB messages and ROS 2 messages and types, effectively allowing direct access to PX4 from ROS2 workflows and nodes.
+The bridge uses UORB message definitions and correspondent IDL types to generate code to serialise and deserialise the messages heading in and out of PX4.
```

Comment:
@TSC21 I subedited this whole section for readability. The new text captures the main points, possibly throwing away some detail I thought was unnecessary but you may think is essential. 

For example this sentence. Please read it (the whole section) carefully and make sure it isn't "wrong" and captures the essentials!

Reply:
Yep these look much better! Thanks!